engineering/conventions.md

```diff
@@ -47,6 +49,10 @@ We have additional conventions for each language:
 * Use latest [alpine](https://hub.docker.com/_/alpine/) or [busybox](https://hub.docker.com/_/busybox/) whenever it is possible. Use latest [debian](https://hub.docker.com/_/debian/) stable-slim otherwise.
 * If needed to reduce image size, use [multi-stage builds](https://docs.docker.com/develop/develop-images/multistage-build/).
 
+## Error Handling
+
+* If it is natural in your language, prefer early handling of errors and return early (more info: [here](http://blog.timoxley.com/post/47041269194/avoid-else-return-early), [here](http://wiki.c2.com/?GuardClause) and [here](http://wiki.c2.com/?HandleErrorsInContext)).
```

Comment:
s/natural/idiomatic/

Reply:
I'd replace the `here` by something more meaningful.

More info:
- [Avoid else, return early](http://blog.timoxley.com/post/47041269194/avoid-else-return-early)
etc